pkg/command/context.go

```diff
@@ -1150,6 +1151,124 @@ func getValues(m map[configtypes.ContextType]*configtypes.Context) []*configtype
 	return values
 }
 
+func newCurrentCtxCmd() *cobra.Command {
+	var currentCtxCmd = &cobra.Command{
+		Use:               "current",
+		Short:             "Display the current context",
+		Args:              cobra.NoArgs,
+		ValidArgsFunction: noMoreCompletions,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			currentCtxMap, err := config.GetAllActiveContextsMap()
+			if err != nil {
+				return err
+			}
+
+			if len(currentCtxMap) == 0 {
+				fmt.Fprintln(cmd.OutOrStdout(), "There is no active context")
+				return nil
+			}
+
+			ignoreTMCCtx := false
+			if len(currentCtxMap) > 1 {
+				// If there are multiple contexts, which means 2 of them, ignore the TMC context
+				// and prioritize the tanzu or k8s context (only one of those two will be present)
+				ignoreTMCCtx = true
+			}
+
+			for ctxType, ctx := range currentCtxMap {
+				if ignoreTMCCtx && ctxType == configtypes.ContextTypeTMC {
+					continue
+				}
+
+				if shortCtx {
+					printShortContext(cmd.OutOrStdout(), ctx)
+				} else {
+					printContext(cmd.OutOrStdout(), ctx)
+				}
+			}
+			return nil
+		},
+	}
+
+	currentCtxCmd.Flags().BoolVarP(&shortCtx, "short", "", false, "print a short form of the context suitable for prompts")
```

Comment:
Not sure we need to presume how this form might be used.
Maybe just "prints the context in compact form" is sufficient?

That said, an example of how it could be used for shell prompt might be worth mentioning in the markdown documentation.

Reply:
I updated the flag description.
As for documenting the prompt, since the format depends on the shell and some user choices (for example, I use `RPROMPT` for this in `zsh` and not the main `PROMPT`), I simply put a note in the `quickstart.md` file about the existence of `tanzu context current --short`